toucan_connectors/postgres/postgresql_connector.py

```diff
@@ -169,17 +175,40 @@ def describe(self, data_source: PostgresDataSource):
             res = cursor.description
         return {r.name: types.get(r.type_code) for r in res}
 
-    def get_model(self):
+    def get_model(self) -> Tuple[List[TableInfo], Dict]:
         """Retrieves the database tree structure using current connection"""
+        available_dbs = self._list_db_names()
+        databases_tree = []
+        for db in available_dbs:
+            with suppress(pgsql.OperationalError):
+                databases_tree += self._list_tables_info(db)
+        return DiscoverableConnector.format_db_model(databases_tree)
+
+    def get_model_with_info(self) -> Tuple[List[TableInfo], Dict]:
+        """Retrieves the database tree structure using current connection"""
+        available_dbs = self._list_db_names()
+        databases_tree = []
+        failed_databases = []
+        for db in available_dbs:
+            try:
+                databases_tree += self._list_tables_info(db)
+            except pgsql.OperationalError:
+                failed_databases.append(db)
+
+        tables_info = DiscoverableConnector.format_db_model(databases_tree)
+        metadata = dict()
+        if len(failed_databases):
+            metadata["info"] = {"Could not reach databases": failed_databases}
```

Comment:
this repo uses single quotes if I'm not mistaken. Haven't checked the linter

Reply:
There is something wrong with my linter. I'll check that.

Reply:
yep that's configured in `setup.cfg`